var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -75,6 +76,7 @@ class NetcdfC(AutotoolsPackage):
     variant("dap", default=False, description="Enable DAP support")
     variant("jna", default=False, description="Enable JNA support")
     variant("fsync", default=False, description="Enable fsync support")
+    variant("zstd", default=False, description="Enable ZStandard compression", when="@4.9.0:")
```

Comment:
I think it would be better to enable zstd by default. It should be supported everywhere.

Reply:
Ok, that's fine with me. Now we only need to find a way to update the PR!